src/ports/rt-kernel/lldp-mib.c

```diff
@@ -0,0 +1,1433 @@
+/*
+Generated by LwipMibCompiler
+*/
+
+#include "pf_includes.h"
```

Comment:
Do we need to include all this? Would pf_snmp.h and pf_types.h be sufficient?

Reply:
The following is sufficient:
#include "options.h"
#include "osal.h"
#include "osal_log.h"
#include "pnal.h"
#include "pnet_api.h"
#include "pf_types.h"

Reply:
OK, keep it as-is. I just would like to avoid using arbitrary functions from the entire stack.